features/registry/registry.feature

```diff
@@ -80,23 +80,22 @@ Feature: Testing registry
       | resource_name | image-registry |
       | o             | yaml           |
     Then the step should succeed
-    And the output should contain:
-      | name: system:registry                      |
-      | resource: clusterroles                     |
-      | name: registry-registry-role               |
-      | resource: clusterrolebindings              |
-      | name: registry                             |       
-      | resource: serviceaccounts                  |
-      | name: image-registry-certificates          |
-      | resource: configmaps                       |
-      | name: image-registry-private-configuration |
-      | resource: secrets                          | 
-      | name: image-registry                       |
-      | resource: services                         |
-      | name: node-ca                              |
-      | resource: daemonsets                       |
-      | name: image-registry                       |
-      | resource: deployments                      |
+    And the output should match:
+      | name: system:registry                         |
+      | resource: clusterroles                        |
+      | name: registry-registry-role                  |
+      | resource: clusterrolebindings                 |
+      | (name: registry)?                             |       
+      | (resource: serviceaccounts)?                  |
+      | (name: image-registry-certificates)?          |
+      | (resource: configmaps)?                       |
+      | (name: image-registry-private-configuration)? |
+      | (resource: secrets)?                          | 
+      | (name: image-registry)?                       |
+      | (resource: services)?                         |
+      | (name: node-ca)?                              |
+      | (resource: daemonsets)?                       |
+      | (resource: deployments)?                      |
```

Comment:
No matter those resources exist or not, the scenario will pass when using like above. Then what's the meaning of leaving them there ?

Reply:
some versions it's not include these, but older versions will had these.